scripts/apps/authoring-react/authoring-section/get-field-container.tsx

```diff
@@ -45,7 +45,12 @@ export function getFieldContainer(
                             style={{minHeight: 'auto'}}
                         >
                             <Spacer h gap="8" noGrow>
-                                <span>{field.name}</span>
+                                <span>
+                                    {field.fieldConfig.required && (
+                                        <span style={{color: '#e41b21', fontSize: '1.2rem'}}>* </span>
```

Comment:
Please don't use any hardcoded values for colors and font-sizes. CSS variables should be used instead.

Reply:
Which variable should be used for this color?

Reply:
@fritzSF did you find the variable yet or you want us to use the hardcoded one? :laughing:

Actually we did try to find the variable in the beginning as well, but couldn't. I think we need some cleanup on colors so we have less of them and they're more descriptive like `color-text-error` vs `red-600`

Reply:
never mind, @thecalcc says you replied to him personally